core/mondoo-email-security.mql.yaml

```diff
@@ -47,7 +47,35 @@ policies:
           - uid: mondoo-email-security-spf-fail
           - uid: mondoo-email-security-spf-dns-record
           - uid: mondoo-email-security-dkim
+          - uid: mondoo-email-security-reverse-ip-ptr-record-set
 queries:
+  - uid: mondoo-email-security-reverse-ip-ptr-record-set
+    title: Ensure Reverse IP Lookup PTR record is set (DNS Forward confirmed)
+    mql: |
+      reverseDNSDomain =
+        dns.params.A.rData.first.split(".")[3] + "."
+          + dns.params.A.rData.first.split(".")[2] + "."
+          +  dns.params.A.rData.first.split(".")[1] + "."
+          +  dns.params.A.rData.first.split(".")[0]
+          + ".in-addr.arpa"
+      dns(reverseDNSDomain).params.PTR.rData.first.contains(asset.fqdn)
+    docs:
+      desc: |
+        Reverse DNS queries for IPv4 addresses use the special domain in-addr.arpa. In this domain, the IPv4 address is represented as a concatenated sequence of four decimal numbers separated by periods, to which is added the second-level domain suffix .in-addr.arpa. The four decimal numbers are obtained by splitting the 32-bit IPv4 address into four octets and converting each octet to a decimal number. These decimal numbers are then arranged in the following order: the smallest octet is first (leftmost) and the most significant octet is last (rightmost). It is important to note that this is the reverse order of the usual convention for writing IPv4 addresses in text form.
+
+        For example, to query the IP address 8.8.4.4 in reverse order, the PTR record for the domain name 4.4.8.8.in-addr.arpa is queried, which points to dns.google.
+
+        If the A record for dns.google in turn points back to 8.8.4.4.4, it will be said to be forward confirmed.
```

Comment:
"... it will be said to be forward confirmed" is tough  to understand. Is "forward confirmed" a state? Who or what says it's foward confirmed? 
It would be great if this sentence could use a structure more like "If the A record for dns.google points back to 8.8.4.4.4, <something> confirms that it's <somethingelse>."

Reply:
Thanks for the note, @misterpantz .
Added some more text and explanations.